packages/@aws-cdk/core/lib/cfn-parse.ts

```diff
@@ -179,6 +179,42 @@ function parseIfCfnIntrinsic(object: any): any {
       const value = parseCfnValueToCdkValue(object[key]);
       return Fn.join(value[0], value[1]);
     }
+    case 'Fn::Cidr': {
+      const value = parseCfnValueToCdkValue(object[key]);
+
+      if (value.length === 2) {
+        return Fn.cidr(value[0], value[1]);
+      }
+      return Fn.cidr(value[0], value[1], value[2]);
+    }
+    case 'Fn::FindInMap': {
+      const value = parseCfnValueToCdkValue(object[key]);
+      return Fn.findInMap(value[0], value[1], value[2]);
+    }
+    case 'Fn::Select': {
+      const value = parseCfnValueToCdkValue(object[key]);
```

Comment:
`Fn.select()` takes a number as the first argument, not a string, so you might consider converting the string you got to a number.

Reply:
Actually. Turns out, CloudFormation accepts both. So feel free to leave this as-is (maybe make sure we have both `{ "Fn::Select": [0, ... }` and `{ "Fn::Select": ["0", ... }` in the tests)